### ISSUE TITLE
obsolete?

### ISSUE THREAD

**Issue by ajs6f** [Smithsonian/info#1]
This overview seems a good bit out of date-- is it reliable?

**Comment by MikeTrizna** [comment]
I agree that this repo looks very out of date, and the only committer @andrewcstewart no longer works at the Smithsonian.

It looks like it was an attempt to organize the various repos in the Smithsonian organization by the specific group that controlled them. I agree that this is useful, but this can also be done by using unique GitHub Topic tags, but only approved committers can add them.

(Here's an example of what this looks like:
![2018-03-22_10-38-42](https://user-images.githubusercontent.com/472677/37777173-5050dcee-2dbd-11e8-8d79-80ddadb52d87.png)

**Comment by ajs6f** [comment]
I'm inclined to suggest that we make this repo private (to SI) if it is unmaintained and out-of-date and (as @MikeTrizna rightly points out) there are better ways to do this kind of organizational work. We have tools like teams, tags, etc. for this.

**Comment by SmithersA** [comment]
I wanted to see if this could be relevant to an ongoing effort for "One Smithsonian"

**Comment by ajs6f** [comment]
If by chance there is some useful info in here for that purpose, it will still be available if I make this private. I'll wait another day or so (just in case anyone else has a comment) and then so do.

Here's another question: I see a link in this repo to `http://ossdev.si.edu/`, which resolves to an unstyled, content-free SI website. Anyone know what that was all about?

**Comment by ajs6f** [comment]
Incidentally, @SmithersA, just for info, what _was_ OCIO's "Office of Research Information Services" is now OCIO's "Research Computing" (I think we are still an "Office", but can't remember off the top of my head), and we are working on our new website. I can check on that, if you are curious.

**Comment by SmithersA** [comment]
@ajs6f Could you email me more about the other tools you have for accomplishing what the person, andrewcstewart, started? I was wondering if an attempt toward something exactly like this (https://github.com/18F/usds-hub/blob/master/README.md) should be made

**Comment by ajs6f** [comment]
@SmithersA Sure, but the truth is that you'll be better served by the Github documentation. If I understood @MikeTrizna correctly, and certainly this is what I think, rather than inventing (or even adopting) an external tool to organize our GH presence, we should just use the baked-in Github tools. They include (but are not limited to) [topics](https://help.github.com/articles/about-topics/) and [teams](https://help.github.com/articles/organizing-members-into-teams/).

**Comment by ajs6f** [comment]
@SmithersA To be clear, I'm not in any way against using cool tools like what 18F has done-- I'm just not sure we're talking about the same thing. The way I'm seeing it, there's "organizing ourselves in Github" and there's "presenting a clear view to ourselves of our software work" and the former feeds the latter.

**Comment by SmithersA** [comment]
@ajs6f Sounds good. Are there 19 teams in GH like there are 19 museums?

**Comment by ajs6f** [comment]
😆  No, no there are not: https://github.com/orgs/Smithsonian/teams 😭 

I am _all_ in favor of developing some strong mapping between our Github presence and the way SI is actually built up out of units and central services. What has happened up to now has been very organic, pushed forward by folks like @ddavis and early adopters. 

I wouldn't want to restrict people from forming their own teams (as you can see they have very successfully done), because that's really useful (and you can key actions and permissions to and have discussions with teams). But having some set baseline practice (i.e. everyone who works for unit X is automagically a member of the team for Unit X, maybe others, but at least that) would make a much clearer picture.

Part of the issue here is that there isn't even a clear mapping between Github identities and SI people. We haven't ever managed [to get SSO](https://help.github.com/articles/managing-member-identity-and-access-in-your-organization-with-saml-single-sign-on/) working and I'm not sure we can with our current licensing arrangements  (@ddavis, feel free to add any needful info). It's also questionable whether we would _want_ to do that-- we have lots of folks (I'm one!) who exist in both the more regulated world of software at SI and other more free-form worlds (e.g. I am an Apache committer, and practice there is radically different than at SI, as it should be).

Ideas welcome!

**Comment by ddavis** [comment]
It is wonderful to see this thread.  It would be great to get the active users of the SI Github to determine how things should be done.  We can use the Wiki and notifications from Github or use the SI Confluence for more privacy until we want to unleash our ideas on the world.  I took a best guess for settings in Github to get a critical mass of user and to permit private SI activities while permitting collaboration with outside folks.  And meeting our security requirements.  And permitting integrations.

Any takers to formalize this community a bit more (while keeping it as lightweight as possible)?

**Comment by ajs6f** [comment]
That's a good (hard) question. If every SI person from a given unit was on a team for that unit, would it answer your question to be able to see what a given team (and therefore people from that unit) is doing on Github? Is that kind of what you are looking for? Keep in mind that lots of projects (in fact most of the projects I work on) have folks working on them from several different units...

All we can ever get out of Github is what people are doing on Github, but maybe that is worth doing? There are also tools with high integration to Github, like Gitter.im or Slack, but they can be problematic because of their record-keeping and privacy polices.

**Comment by ajs6f** [comment]
A person you will definitely want to talk to is @edwardmonk, in OCIO. He is working on (amongst much else) the problem of sharable modules for front-end websites. Drupal, specifically, but the point is general.

I'm not honestly not quite sure what an API would do here-- it seems (at first glance) to me like a problem that would be solved by lots and lots of person-to-person communication more than automation. 

When that person asked you about getting "something similar to Air & Space", were they not also able to put in you in touch with their developer(s)? Could that person (or persons) not tell you how their site is built?

**Comment by SmithersA** [comment]
Okay, thanks!

It seemed like they were scared to talk. I am fairly new still so I don't know if they've been burned in the past or if it wasn't the agency culture to "talk".

**Comment by ddavis** [comment]
You are not the first with these observation.  There isn't a single resource for information within SI.  And no well formed community of developers. Folks tend towards privacy. But we are trying to get better. Github is part of this. And its been easier to get folks to provide information and documentation in SI's Confluence and JIRA with more formal stuff in Prism.  But its early days.  It sounds like you want to work toward more sharing.

**Comment by SmithersA** [comment]
Yes (and I want everyone to feel safe to do so and enjoy it)

**Comment by ajs6f** [comment]
It's definitely not the agency culture "not to talk", but SI is anything but a centralized place and every unit has its own "personality". I haven't ever refrained from asking questions or trying to get people to talk and I have yet to have anything but good come of it.

**Comment by SmithersA** [comment]
@ajs6f @ddavis Okay! These are good things to hear. I will definitely keep poking around and try to make something happen :-) 
I guess I should start with a list of names and find out who's who

**Comment by ddavis** [comment]
Anyone who wants more information about what is currently in progress regarding how we approaching information and documentation right now, please contact me. Most source code is being kept here in Github some in public repos and much in private-to-SI repos.  I can tell you what we have so far towards a better open, sharing approach.

**Comment by SmithersA** [comment]
@ddavis Are there any custom search repos (outside of NMAAHC)?

**Comment by ajs6f** [comment]
I don't know the specifics of the group in question, but if you are comfortable sharing them, we can certainly try to help find you an interlocutor. One of the nice things about working at OCIO (as @ddavis and I both do) is that you meet people from all over the Institution!

**Comment by ajs6f** [comment]
@SmithersA @ddavis It's worth pointing out that (as far as I know) we don't actually have any policy for information and documentation. We have several instances of Confluence and Jira, we have Github (obviously!) and we have any number of older or home-grown tools as well.

**Comment by ddavis** [comment]
@SmithersA You said repos so I may answer the question wrong. EDAN is the big one. But there are many of them one or more per unit.  Most museums have at least one central one.  The EDAN community group fairly new but is a good place to start.  They are building a Confluence site but just beginning. So you may have to poll the members.

**Comment by ajs6f** [comment]
@SmithersA 

If, instead, you mean _code_ repos, then again, @edwardmonk is potentially a person to talk to, because his group is working on Drupal modules that act as clients for EDAN (the centralized index to which @ddavis just referred).

If, instead, you are thinking of _site_ search (as opposed to searching the _collections_, which is what EDAN does), then I believe Google's site search is the supported tool, and I'm not sure whether there are centrally supported tools for unit websites to use for that, but we can find out.

Maybe you can clarify what you are looking for?

**Comment by SmithersA** [comment]
definitely site search

**Comment by ajs6f** [comment]
Hey, @SmithersA -- again, @edwardmonk is going to be a good place to start asking about that. We certainly have lots of different flavors of site search presentation. I don't deal with that myself, so I don't know how much coordination is already present, but if nothing else, you should be able to get talking to some other site search maintainers.

**Comment by ajs6f** [comment]
Since no one has objected, I will switch this code repo to private.
